dvc/fs/s3.py

```diff
@@ -136,6 +137,7 @@ def s3(self):
         return session.resource(
             "s3",
             endpoint_url=self.endpoint_url,
+            verify=self.verify_ssl,
```

Comment:
For the record, we are migrating to s3fs right now, and looks like we will be able to just use `client_kwargs` there https://github.com/dask/s3fs/blob/main/s3fs/core.py